packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -245,3 +228,7 @@ export default class SaytDriverPlugin implements Plugin {
       .map((term) => ({ label: term.value }));
   }
 }
+
+export interface SaytDriverOptions {
+  productTransformer?: ProductTransformer<Product>;
```

Comment:
Add docs:

```ts
  /**
   * A function to transform a [[Record]] from the GroupBy API to the
   * desired form.
   */
```

This should also use the generic from the interface:

```suggestion
  productTransformer?: ProductTransformer<P>;
```